modules/efk-logging-deploying-about.adoc

```diff
@@ -0,0 +1,220 @@
+// Module included in the following assemblies:
+//
+// * logging/efk-logging-deploying-about.adoc
+
+[id="efk-logging-deploying-about-{context}"]
+= About deploying and configuring cluster logging
+
+{product-title} cluster logging is designed to be used with the default configuration that should support most {product-title} environments. 
```

Comment:
i wouldn't mention "dedicated" clusters (not clear to me what you mean, but to me it means an our online dedicated envs which means nothing to most user anyway), but saying "the default configuration is tuned for small to medium sized clusters" seems reasonable.  (notice the substitution of "tuned for" rather than "should/will support").